plugins/travis-ci/src/api/index.ts

```diff
@@ -58,33 +43,44 @@ export class CircleCIApi {
       limit: number;
       offset: number;
     },
-    token: string, // options: CircleCIOptions,
+    { token, owner, repo }: { token: string; owner: string; repo: string }, // options: CircleCIOptions,
   ) {
-    return (
-      await fetch(`${BASE_URL}builds?limit=${limit}&offset=${offset}`, {
-        headers: { Authorization: token },
-      })
+    const repoSlug = encodeURIComponent(`${owner}/${repo}`);
+
+    const response = await (
```

Comment:
Eventually, we will probably need some error handling for when these requests fail. It probably wouldn't live in this file, since we would want to inform the user about the error, but something to keep in mind.